server/views/partials/footer.njk

```diff
@@ -12,18 +12,5 @@
 {% include "../components/feedback-banner.njk" %}
 
 {{ 
-  govukFooter({
-    meta: {
-      items: [
-        {
-          href: '/feedback-and-support',
-          text: "Get help",
-          attributes: {
-                'data-qa': 'get-help'
-            }
-        }
-      ]
-    },
-    classes: "govuk-!-display-none-print qa-court-service-footer"
-  }) 
+  govukFooter() 
```

Comment:
Just checking, do we need this now? Does the gov footer not show at all if we don't add this? Just checking as we could see the footer previously without it but other changes may mean it's now necessary.